.github/workflows/availability-tests.yml

```diff
@@ -0,0 +1,190 @@
+---
+name: Availability Tests
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: The environment to run against
+        required: true
+        type: string
+      rpc_url:
+        description: JSON-RPC URL
+        required: true
+        type: string
+      fund_amount:
+        description: Amount in Ether to fund
+        required: true
+        type: string
+        default: "5"
+      london:
+        description: Is London fork active?
+        required: true
+        type: boolean
+        default: true
+      notification:
+        description: Notification
+        type: boolean
+        default: true
+  workflow_call:
+    inputs:
+      environment:
+        description: The environment to run against
+        required: true
+        type: string
+      rpc_url:
+        description: JSON-RPC URL
+        required: true
+        type: string
+      fund_amount:
+        description: Amount in Ether to fund
+        required: true
+        type: string
+      london:
+        description: Is London fork active?
+        required: true
+        type: boolean
+      notification:
+        description: Notification
+        type: boolean
+        required: true
+    outputs:
+      availability_test_status:
+       value: ${{ jobs.availability_test.outputs.availability_test_status }}
+      produce_blocks:
+        value: ${{ jobs.availability_test.outputs.produce_blocks }}
+      fund_status:
+        value: ${{ jobs.availability_test.outputs.fund_status }}
+      check_deployed_smart_contract_status:
+        value: ${{ jobs.availability_test.outputs.check_deployed_smart_contract_status }}
+      method_set_status:
+        value: ${{ jobs.availability_test.outputs.method_set_status }}
+      method_get_status:
+        value: ${{ jobs.availability_test.outputs.method_get_status }}
+    secrets:
+      FAUCET_PRIVATE_KEY:
+        required: true
+      ACCOUNT_PRIVATE_KEY:
+        required: true
+      ACCOUNT_PUBLIC_ADDRESS:
+        required: true
```

Comment:
If this is the address of the same account that `ACCOUNT_PRIVATE_KEY` denotes, then you can derive its address using Foundry (https://book.getfoundry.sh/reference/cast/cast-wallet-address):

```bash
cast wallet address --private-key <RAW_PRIVATE_KEY>
```

Reply:
Is it safe to expose our Public Address? As it stands now, the Public Address will remain hidden, and no one will be able to see it. However, since it's a Public Address, it technically shouldn't matter if it's visible. WTYT?

Reply:
Yes, public address (as its name suggests) is publicly available data and it is safe to expose it and to get know by 3rd parties. On the other hand, the private key is the data that must be kept secured. Finally, address can be derived from the private key and that's why I have suggested to remove this parameter **in case ACCOUNT_PUBLIC_ADDRESS and ACCOUNT_PRIVATE_KEY are the same actor(s) in the availability tests**.

Reply:
Changed in [038c922](https://github.com/Ethernal-Tech/blade/pull/429/commits/038c922e37b48d4c4138a5c2ce27d894ac98c716)